src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -155,6 +155,8 @@ DELETE FROM tab_calculation_method_rates WHERE calculation_method='ActiveProcess
 DELETE FROM tab_calculation_method_rates WHERE calculation_method='ActiveProcess_PKSim' AND formula_rate='RelExpPlasmaNormGlobal';
 DELETE FROM tab_calculation_method_rates WHERE calculation_method='ActiveProcess_PKSim' AND formula_rate='RelExpVascEndoGlobal';
 DELETE FROM tab_calculation_method_rates WHERE calculation_method='ActiveProcess_PKSim' AND formula_rate='RelExpVascEndoNormGlobal';
+UPDATE tab_calculation_method_rates SET formula='P_endothelial_l * SA_pls_int > 0 ? (J_iso+alpha_l*Q_lymph)*(1-sigma_l)/(P_endothelial_l*SA_pls_int) : 0' WHERE calculation_method='CalcPecletNumber' AND formula_rate='PecletNumber_l';
+UPDATE tab_calculation_method_rates SET formula='P_endothelial_s * SA_pls_int > 0 ? (-J_iso+(1-alpha_l)*Q_lymph)*(1-sigma_s)/(P_endothelial_s*SA_pls_int) : 0' WHERE calculation_method='CalcPecletNumber' AND formula_rate='PecletNumber_s';

```

Comment:
PEclet number (l/s) were indeed the only parameters which were NaN in pericentral zone when it's inactive.